recipes-wpe/wpeframework/wpeframework_git.bbappend

```diff
@@ -0,0 +1 @@
+PACKAGECONFIG_append = " virtualinput"
```

Comment:
Could you please move static part of PACKAGECONFIG to the .conf file?

Reply:
done